<launch>
  <arg name="camera" default="kinect_head"/>
  <group ns="fridge_cds_stocker">
    <node pkg="roseus" type="roseus" name="fridge_cds_stocker"
          output="screen"
          args="$(find sift_pr2)/node_script/kalman.l">
      <remap from="input/image" to="/$(arg camera)/rgb/image_rect_color" />
      <remap from="input/ObjectDetection" to="/$(arg camera)/rgb/ObjectDetection" />
      <param name="object_width" value="0.231"/>
      <param name="object_height" value="0.196"/>
      <param name="relative_pose" value="0.22 0.05 0.075  0.707107 0
        -0.707107 0"/>
      <param name="target_type" value="fridge" />
    </node>
  </group>
</launch>
